authapi/views.py

```diff
@@ -83,7 +92,7 @@ def get_queryset(self):
         We also have the query params permission_contains and object_id, which
         allow users to filter the teams based on the permissions they
         contain.'''
-        queryset = self.queryset
+        queryset = super(TeamViewSetNoCreate, self).get_queryset()
         if self.action == 'list':
             archived = get_true_false_both(
                 self.request.query_params, 'archived', 'false')
```

Comment:
I see the similarity, but I think naming the view sets to match what resource they are representing just makes it a bit clearer (I needed to read a bit before understanding what resource `TeamViewSetNoCreate` was representing, and how it related to `TeamViewSet`). Also thinking we might come across a case where the viewset for `/organizations/<id>/teams` needs to change in some way, but that change shouldn't affect the viewset for `/teams/`.

Its bikesheddy enough a point that I'm happy with the current approach if you prefer it.